### PR TITLE
[ASUSGAB] Add localization

### PR DESCRIPTION
**Pull request by AndreyLykhoman**
*DESCRIPTION:* Add localization support
*TASK:* https://youtrack.raccoongang.com/issue/ASUSGAB-95

### CODE REVIEW

File: done/done.py [hunk 1]
```diff
@@ -1,37 +1,72 @@
...
-        default="left"
+        help=_("Align left/right/center"),
+        default=_("left")
     )
 
     has_score = True
 
+    loader = ResourceLoader(__name__)
+
+    block_settings_key = 'done'
```
Comment (idegtiarov) [inline done/done.py]: Please point me where block_settings_key and default_theme_config are used?

File: done/done.py [hunk 2]
```diff
@@ -1,37 +1,72 @@
...
+    default_theme_config = {
+        'package': 'done',
+        'locations': ["static/css/done.css"]
+    }
+
+    @staticmethod
+    def resource_string(path):
```
Comment (idegtiarov) [inline done/done.py]: This method is not used. Do we need it?

File: done/translations/config.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Configuration for i18n workflow.
+
+locales:
+    - en  # English - Source Language
+    - pt_BR  # Portuguese (Brazil)
+
+
+# The locales used for fake-accented English, for testing.
+#dummy_locales:
+#    - eo
```
Comment (idegtiarov) [inline done/translations/config.yaml]: Please add the trailing empty line.

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: done/done.py (hunk 2 above)
Comment (AndreyLykhoman) [reply done/done.py]: I deleted it.

File: done/done.py (hunk 1 above)
Comment (AndreyLykhoman) [reply done/done.py]: This code doesn't use those settings. I deleted it.

File: done/translations/config.yaml (hunk 3 above)
Comment (AndreyLykhoman) [reply done/translations/config.yaml]: I added.

**Pull request merged**